skins/cat17/templates/page_layouts/faq.html.twig

```diff
@@ -3,9 +3,12 @@
 {% block page_identifier %}faq{% endblock%}
 
 {% block main %}
-	<div id="faq"></div>
+	<div id="faq" content="{$ faq_content $}"></div>
 {% endblock %}
 
 {% block scripts %}
+	<script type="text/javascript">
+		var faqContent = JSON.parse( document.getElementById( "faq" ).getAttribute( "content" ) );
```

Comment:
This creates a global variable that will trigger ESLint on `faq.js` I think it would be better to move the call there.

Reply:
but that's in the `web` folder :fearful:
Do you mean the `main.js` in the `faq-page` folder ?